### PR TITLE
feat: StabilityAI DLC Image URIs

### PR DESCRIPTION
**Pull request by amzn-choeric**
*Issue #, if available:*

*Description of changes:*
Adding in the image URI configurations for the new StabilityAI DLC inference image. 

Note that there is currently no fine-tuning container, and that it has not been determined if we wil have a dedicated model class to fetch the image URIs. 

*Testing done:*
Ran unit tests.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I certify that the changes I am introducing will be backward compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/image_uri_config/stabilityai.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    "ap-northeast-3": "364406365360",
+                    "ap-south-1": "763104351884",
+                    "ap-south-2": "772153158452",
+                    "ap-southeast-1": "763104351884",
+                    "ap-southeast-2": "763104351884",
+                    "ap-southeast-3": "907027046896",
+                    "ap-southeast-4": "457447274322",
+                    "ca-central-1": "763104351884",
+                    "cn-north-1": "727897471807",
+                    "cn-northwest-1": "727897471807",
```
Comment (arjkesh) [inline src/sagemaker/image_uri_config/stabilityai.json:25]: we should remove unsupported regions (cn/gov) for now

File: tests/unit/sagemaker/image_uris/test_stabilityai.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    "ap-east-1": "871362719292",
+    "ap-northeast-1": "763104351884",
+    "ap-northeast-2": "763104351884",
+    "ap-northeast-3": "364406365360",
+    "ap-south-1": "763104351884",
+    "ap-southeast-1": "763104351884",
+    "ap-southeast-2": "763104351884",
+    "ap-southeast-3": "907027046896",
+    "ca-central-1": "763104351884",
+    "cn-north-1": "727897471807",
```
Comment (arjkesh) [inline tests/unit/sagemaker/image_uris/test_stabilityai.py:32]: we should remove unsupported regions (cn) for now

**Review by arjkesh — COMMENTED** [review]
outside of regions this lgtm

**Review by knikure — COMMENTED** [review]
/bot run all

**Review by arjkesh — APPROVED** [review]
lgtm

**Review by knikure — APPROVED** [review]

**Pull request merged**